src/features/utils/lazy-feature-loader.js

```diff
@@ -24,8 +24,6 @@ export function lazyFeatureLoader (featureName, featurePart) {
         return import(/* webpackChunkName: "logging-aggregate" */ '../logging/aggregate')
       case FEATURE_NAMES.metrics:
         return import(/* webpackChunkName: "metrics-aggregate" */ '../metrics/aggregate')
-      case FEATURE_NAMES.pageAction:
-        return import(/* webpackChunkName: "page_action-aggregate" */ '../page_action/aggregate')
```

Comment:
This line should remain while we still have the `features/page_action` feature exported. Instead, change this to just import the `generic_events` aggregate.

```
case FEATURE_NAMES.pageAction:
        return import(/* webpackChunkName: "generic_events-aggregate" */ '../generic_events/aggregate')
```

Reply:
nvm